eng/versioning/version_client.txt

```diff
@@ -5,14 +5,14 @@ com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-core;1.1.0;1.2.0
+com.azure:azure-core;1.2.0;1.3.0-beta.1
 com.azure:azure-core-amqp;1.0.0-beta.8;1.0.1
-com.azure:azure-core-http-netty;1.1.0;1.2.0
-com.azure:azure-core-http-okhttp;1.0.0;1.1.0
+com.azure:azure-core-http-netty;1.2.0;1.3.0-beta.1
+com.azure:azure-core-http-okhttp;1.1.0;1.2.0-beta.1
 com.azure:azure-core-management;1.0.0-beta.8;1.0.0-beta.8
-com.azure:azure-core-test;1.1.0-beta.2;1.1.0
-com.azure:azure-core-tracing-opencensus;1.0.0-beta.5;1.0.0-beta.6
-com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-core-test;1.1.0;1.1.1-beta.1
```

Comment:
Hemant, with 1.1.0 being just released, the azure-core-test's current version should be 1.2.0-beta.1 to keep it in line with the other version increments.